From ef0c57fe47248ed482ee3910dd8ecb5b70d4e033 Mon Sep 17 00:00:00 2001
From: "kirk.wang" <kirk.wang@amlogic.com>
Date: Thu, 3 Aug 2023 20:14:03 +0800
Subject: [PATCH] t7: delete t7c 5.15 conf [1/1]

PD#SWPL-131298

Problem:
delete t7c 5.15 conf

Solution:
the t7 configuration file is compatible with t7c

Verify:
t7c/t7b

Signed-off-by: kirk.wang <kirk.wang@amlogic.com>
Change-Id: I2fc521aec0abcc8d29f66ccbfc3b5003ba26ab16
---
 conf/machine/mesont7c-an400-5.15.conf         |  1 -
 .../machine/mesont7c-an400-weston-5.15.config | 95 -------------------
 2 files changed, 96 deletions(-)
 delete mode 120000 conf/machine/mesont7c-an400-5.15.conf
 delete mode 100755 conf/machine/mesont7c-an400-weston-5.15.config

diff --git a/conf/machine/mesont7c-an400-5.15.conf b/conf/machine/mesont7c-an400-5.15.conf
deleted file mode 120000
index aeb9fe2..0000000
--- a/conf/machine/mesont7c-an400-5.15.conf
+++ /dev/null
@@ -1 +0,0 @@
-mesont7c-an400-weston-5.15.config
\ No newline at end of file
diff --git a/conf/machine/mesont7c-an400-weston-5.15.config b/conf/machine/mesont7c-an400-weston-5.15.config
deleted file mode 100755
index a35a4a9..0000000
--- a/conf/machine/mesont7c-an400-weston-5.15.config
+++ /dev/null
@@ -1,95 +0,0 @@
-#@TYPE: Machine
-#@NAME: meson
-
-#@DESCRIPTION: Machine configuration for meson systems
-
-include conf/machine/include/mesont7c_k5.15_64b.inc
-include conf/machine/include/amlogic_weston.inc
-
-PREFERRED_PROVIDER_virtual/mesa = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/mesa-gl = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgbm = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/egl = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgl = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgles1 = "libgles-eabihf-gondul-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgles2 = "libgles-eabihf-gondul-wayland-drm"
-
-MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
-        gpu \
-        wififw-ap6398 \
-        ap6xxx \
-        adla \
-        "
-
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " aml-subtitleserver"
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " gst-aml-drm-plugins"
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " fuse-exfat"
-RDEPENDS_packagegroup-amlogic-baserootfs_append = " isp isp-dev"
-RDEPENDS_packagegroup-amlogic-baserootfs_append = " cve libcve"
-RDEPENDS_packagegroup-amlogic-baserootfs_append = " npu-common"
-
-
-PREFERRED_VERSION_isp = "5.15"
-PREFERRED_VERSION_ap6xxx = "5.15"
-CHIPSET_NAME = "A311D2"
-KERNEL_DEVICETREE = "t7c_a311d2_an400_linux.dtb"
-UBOOT_MACHINE = "t7_an400_lpddr4x_config"
-KERNEL_MODULE_AUTOLOAD += " dhd"
-KERNEL_MODULE_PROBECONF += "dhd"
-module_conf_dhd = "options dhd firmware_path=../../etc/wifi/6398/ nvram_path=../../etc/wifi/6398/"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-dummy_codec"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_t9015"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-tas5707"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_codec_tl1_acodec"
-KERNEL_MODULE_AUTOLOAD += " meson_ir"
-KERNEL_MODULE_AUTOLOAD += " hci_uart"
-KERNEL_MODULE_PROBECONF += "hci_uart"
-KERNEL_MODULE_AUTOLOAD += "meson-cpufreq"
-
-DISTRO_FEATURES_append = " drm"
-DISTRO_FEATURES_append = " gpt-partition"
-DISTRO_FEATURES_append = " kernel_515"
-
-#DISTRO_FEATURES_append = " onepass"
-DISTRO_FEATURES_append = " secure-u-boot optee "
-#DISTRO_FEATURES_append = " widevine "
-#DISTRO_FEATURES_remove = " alsa "
-DISTRO_FEATURES_append = " alsa "
-DISTRO_FEATURES_append = " pam"
-DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
-DEFAULT_DISTRO_FEATURES += " systemd"
-DISTRO_FEATURES_append = " systemd"
-DISTRO_FEATURES_remove = " sysvinit"
-DISTRO_FEATURES_append = " gstreamer1"
-DISTRO_FEATURES_append = " aml-dtv "
-DISTRO_FEATURES_append = " gst-plugin-venc"
-DISTRO_FEATURES_append = " aml-libjpeg"
-DISTRO_FEATURES_append = " aml-libmultienc"
-DISTRO_FEATURES_append = " aml-libamvenc"
-#DISTRO_FEATURES_append = " playready "
-DISTRO_FEATURES_append = " aml-isp"
-DISTRO_FEATURES_append = " amlogic-tv "
-DISTRO_FEATURES_append = " softap"
-DISTRO_FEATURES_append = " aml-libgdc"
-DISTRO_FEATURES_append = " swupdate"
-DISTRO_FEATURES_append = " recovery"
-DISTRO_FEATURES_append = " mediactrlsrc"
-DISTRO_FEATURES_append = " mediactrlsrc-cam"
-DISTRO_FEATURES_append = " mediactrlsrc-hdmi"
-
-DISTRO_FEATURES_append = " gst-aml-imgproc"
-DISTRO_FEATURES_append = " gst-aml-gfx2d"
-DISTRO_FEATURES_append = " gst-aml-ion-allocator"
-DISTRO_FEATURES_append = " gst-jpeg-swdec"
-
-DISTRO_FEATURES_append = " gst-plugin-overlay"
-DISTRO_FEATURES_append = " gst-plugin-nn"
-DISTRO_FEATURES_append = " detect-library"
-DISTRO_FEATURES_append = " gst-plugin-conv"
-OVERRIDES .= ":an400:t7:onepass"
-MACHINEOVERRIDES .= ":client"
-
-EXTRA_BLUETOOTH_STUFF := "${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth', bb.utils.contains('DISTRO_FEATURES', 'bluez5', ' bluez5', ' bluez4', d), '', d)}"
-EXTRA_BLUETOOTH_STUFF += " bluez-alsa"
-EXTRA_BLUETOOTH_STUFF += " brcm-patchram-plus"
-DISTRO_FEATURES_append = "${EXTRA_BLUETOOTH_STUFF}"
-- 
2.25.1

